src/lib/pages/workflows.svelte

```diff
@@ -11,82 +11,104 @@
+      $workflowsQuery = query;
+    }
+  }
 
   onMount(() => {
     $lastUsedNamespace = $page.params.namespace;
+    if (query) {
+      // Set filters from inital page load query if it exists
+      $workflowFilters = toListWorkflowFilters(query);
+    }
   });
 
+  const errorMessage =
+    'If you have filters applied, try adjusting them. Otherwise please check your syntax and try again.';
+
   const refreshWorkflows = () => {
     $refresh = Date.now();
   };
-
-  onDestroy(() => {
-    const query = $page.url.searchParams.get('query');
-    const parameters = query ? toListWorkflowParameters(query) : {};
-    $workflowsSearch = { parameters, searchType };
-  });
 </script>
 
 <PageTitle
   title={`Workflows | ${$page.params?.namespace}`}
   url={$page.url.href}
 />
-<div class="flex justify-between">
+<div class="mb-2 flex justify-between">
   <div>
     <h1 class="text-2xl" data-cy="namespace-title">
       Recent Workflows
       <NamespaceSelector />
     </h1>
-    <p class="text-sm text-gray-600" data-cy="namespace-name">
-      {$page.params.namespace}
-    </p>
+    <div class="flex items-center gap-2 text-sm text-gray-600">
+      <p data-cy="namespace-name">
+        {$page.params.namespace}
+      </p>
+      <div class="h-2 w-2 rounded-full bg-gray-400" />
+      <p data-cy="workflow-count">
+        {#if !$loading && !$updating}
+          {#if query}
+            Results {$workflowCount?.count ?? 0} of {$workflowCount?.totalCount ??
+              0} workflows
+          {:else}
+            {$workflowCount?.totalCount ?? 0} workflows
+          {/if}
+        {/if}
+      </p>
+    </div>
   </div>
   <div>
     <Button variant="secondary" on:click={refreshWorkflows}
       ><Icon name="retry" /></Button
     >
   </div>
 </div>
-<WorkflowFilters bind:searchType />
-{#if $loading}
-  <Loading />
-{:else if $workflows.length}
-  <Pagination items={$workflows} updating={$updating} let:visibleItems>
-    <WorkflowsSummaryTable>
-      {#each visibleItems as event}
-        <WorkflowsSummaryRow
-          workflow={event}
-          namespace={$page.params.namespace}
-          timeFormat={$timeFormat}
-        />
-      {/each}
-    </WorkflowsSummaryTable>
-  </Pagination>
-{:else}
-  <EmptyState
-    title={'No Workflows Found'}
-    content={errorMessage}
-    error={$workflowError}
-  />
-{/if}
+<Pagination items={$workflows} let:visibleItems>
+  <svelte:fragment slot="action-top-left">
+    <WorkflowAdvancedSearch />
+  </svelte:fragment>
+  <svelte:fragment slot="action-top-center">
+    <WorkflowDateTime />
+  </svelte:fragment>
+  <WorkflowsSummaryTable updating={$loading || $updating}>
+    {#each visibleItems as event}
+      <WorkflowsSummaryRow
+        workflow={event}
+        namespace={$page.params.namespace}
+        timeFormat={$timeFormat}
+      />
+    {:else}
+      <TableRow>
+        <td colspan="5">
+          <EmptyState
+            title={'No Workflows Found'}
```

Comment:
This is the loading screen when navigating to workflows. Wondering if we might want to consider something that more clearly states it's loading and not that there aren't any workflows?

![Screen Shot 2022-10-12 at 10 26 24 AM](https://user-images.githubusercontent.com/15069288/195397952-a7858967-d8cb-4e60-a65c-6b7665c56111.png)

When filters are applied > the user navigates away from workspaces > then goes back to workspaces this is the loading screen (`Namespace undefined is not found`) which could be confusing as well.

![Screen Shot 2022-10-12 at 10 26 44 AM](https://user-images.githubusercontent.com/15069288/195398140-27cb414f-9c92-4883-93c5-9d9410c5214d.png)